renovate.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
We may need to have Renovate added to this repo, along with Codecov.  I'm not sure if that's in the new repo creation docs. 🤔